xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/js/xwiki/usersandgroups/usersandgroups.js

```diff
@@ -93,13 +94,13 @@ window.MSCheckbox = Class.create({
     img.src = this.images[state];
     
     //Update the description of the button for accessibility.
-    require(['xwiki-l10n!my-translation-keys'], function(l10n) {
+    require(['xwiki-l10n!users-and-group-translation-keys'], function(l10n) {
       var alts = [
-        l10n['inherit'],
+        l10n['undefined'],
         l10n['allowed'],
         l10n['denied']
       ];
-      img.alt = alts[this.state];
+      img.alt = alts[this.state] + " . " + l10n.get('changeStateTo', alts[this.nextState()]);
```

Comment:
I meant for the dot to just be a separator between two different semantic units juxtaposed (current state & purpose of the button).
Okay, got it. I changed the translation in the three states to just contain one string that also defines the next state (since this should not change, I guess it can be hard written in the translations. **Should I add a commentary for these entries in the application resource or is that good as is?**
There's a couple needs/preferences for the translation that might get lost otherwise, but I navigated through the resource bundle and could hardly see any comment except those for delimiting sections.
Something like:
### Needs to define current state and then purpose of the button. Should start with the current state name for easier navigation with screen readers.

Reply:
We rarely add comments, yes, but it would be a good idea to change that. Comments above the translation string are displayed in Weblate afaik so this should work.